packages/web-twig/src/Resources/components/Item/__tests__/__snapshots__/itemDisabled.twig.snap.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <title>
+    </title>
+  </head>
+  <body>
+    <button class="Item Item--disabled" aria-selected="false"><span class="Item__label">Item label</span></button>
```

Comment:
ha. If we have `<button>` we could add attribute `disabled` too 🤔

Reply:
added